plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -482,4 +509,91 @@ describe('pg-pool', () => {
       });
     });
   });
+
+  describe('pg metrics', () => {
+    let otelTestingMeterProvider;
+    let inMemoryMetricsExporter: InMemoryMetricExporter;
+
+    function initMeterProvider() {
+      otelTestingMeterProvider = new MeterProvider();
+      inMemoryMetricsExporter = new InMemoryMetricExporter(
+        AggregationTemporality.CUMULATIVE
+      );
+      const metricReader = new PeriodicExportingMetricReader({
+        exporter: inMemoryMetricsExporter,
+        exportIntervalMillis: 100,
+        exportTimeoutMillis: 100,
+      });
```

Comment:
Thanks for the info, I'll take a look into it!

Reply:
Btw, I copied this implementation from both mongo and mysql tests, since I'm changing on a similar place. Should all these places be update to use the approach you mentioned then?